### ISSUE TITLE
MSC 5.1 and 4.8 Do Not Accurately Reflect Current Munki Status

### ISSUE THREAD

**Issue by splemp** [munki/munki#925]
I've noticed that if a `managedsoftwareupdate --auto` run is currently in progress **before preflight is complete** and a user opens MSC, it does not show that an update is running in the GUI until it starts actually checking for updates. There appears to be something that triggers the GUI to update once it gets past the preflight stage. If a user selects "Check Again" while an update is already in progress (but MSC doesn't show it's in progress), then multiple log messages appear periodically stating "Another instance of managedsoftwareupdate is running as pid xxx." Once the auto run is done, the Manual Check requested by clicking "Check Again" will run. 

Opening MSC in the middle of an auto run once it is past the preflight stage _does_ result in expected behavior where the Updates tab shows an update in progress.

I tested with MSC 5.1.0.3758 and 4.8.3628. 

**Steps to re-produce:**
1) Run `managedsoftwareupdate` via Terminal to simulate a regular background run
    - `--manualcheck` and `--auto` both experience this issue
2) Open Managed Software Center.app before the preflight stage is complete
3) Navigate to Updates tab

**Expected behavior:**
Managed Software Center.app > Updates should indicate an update is running

**Actual behavior:** 
Managed Software Center.app does not reliably indicate that an update is running, allowing users to select "Check Again" when an update is currently in progress.

**Impact:**
Minor. Maybe I should just cleanup what runs at preflight. I'm not sure how this interacts when using middleware.

**Comment by gregneagle** [comment]
I don't think this is an issue we can address such that it does the right thing for all permutations.
One of the things that can happen in preflight is to decide to _not_ run the actual updatecheck and/or install/remove session. So from one point-of-view, a Munki session is _not_ running until after we return from preflight.

**Comment by gregneagle** [comment]
Closing as "won't fix"
